Blazor/README.md

```diff
@@ -19,20 +19,20 @@ Issues can be reported on the [Github repository](https://github.com/wet-boew/cd
 
 To use, in your project: 
 
-  - Add the GoC.WebTemplate.Blazor nuget package to your Blazor project. This should automatically pull in the GoC.WebTEmplate.Entities package as well.
+  - Add the GoC.WebTemplate.Blazor nuget package to your Blazor project. This should automatically pull in the GoC.WebTEmplate.Entities dll as well.
```

Comment:
I'm not sure we should mention the Entities DLLs here.  My concern is that they'll expect a Entities nuget package and wonder why they're not seeing it.  What do you think about "This should automatically pull in the needed DotNetTemplate libraries as well"

Reply:
Yup, I agree, I changed it